src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -84,55 +84,68 @@ We also have [template consent materials](https://docs.google.com/document/d/1il
+We provide the information and consent sheet when the potential participants are invited to take part in your research. And again when each participant is booked in, and we are sending them details of their specific session.
 
-Check which format and channel works best for them - email attachment, paper letter, easy read, verbally, etc.
+We check which format and channel works best for them - email attachment, paper letter, easy read, verbally, etc.
 
 ### Collecting a record of informed consent
 
-You will need to collect a record of their informed consent from each participant, and keep it along with their research data.
+We need to collect a record of their informed consent from each participant, and keep it along with their research data.
 
-For user research sessions where you will not meet the participant face to face, you can:
+For user research sessions where we will not meet the participant face to face, we can:
 
 * ask the participant to reply by email confirming their informed consent
 * send a paper consent form in the post for the participant to sign and return
 
-For user research sessions where you meet the participant in person, you can also:
+For user research sessions where we meet the participant in person, we can also:
 
 * give the participant a printed copy of the consent form to sign
 * share a consent form in an accessible digital format the participant can read and respond to using their assistive technology
 * read the consent form aloud and collect their consent on an audio or video recording
 
-Whichever approach you are taking, make sure the participants have time to read the research and consent information carefully, and that you give them an explicit opportunity to ask questions.
+Whichever approach we are taking, we make sure the participants have time to read the research and consent information carefully. And we give them an explicit opportunity to ask questions.
 
 ### During the session
 
-At the start of the session:
+At the start of the session we:
 
 * remind the participant that their participation is voluntary and they can stop or take a break at any time
-* remind them about the different ways you are recording the session, and about anyone observing or listening in
+* remind them about the different ways we are recording the session, and about anyone observing or listening in
 * ask them to confirm their consent for the recording
 
-As the session continues, actively make sure you have the participant’s consent. For example, check in with the participant if they look concerned about an activity, or if you’re on a visit and want to take a photo in a new area.
+As the session continues, we actively make sure we have the participant’s consent. For example, by checking in with the participant if they look concerned about an activity, or if asking for permission when we’re on a visit and want to take a photo in a new area.
```

Comment:
"if they look" is ableist, it assumes all participants will look the same when concerned and all participants will consent to having their video on or that all sessions will have a visual of the participant's face.

Better we simplyfy this to the core action of "checking if the participant is concerned about an activity".

Here i would expect each user researcher to evaluate this by asking them verbally or in written form or in a form that is accessible to the participant.

And lastly, "checking in with the participant" vs "checking with the participant" - checking in is applicable to checking in into a hotel/flight. https://dictionary.cambridge.org/dictionary/english/check-in

Reply:
OK. Thanks for spotting that.